twake/backend/node/config/default.json

```diff
@@ -9,11 +9,15 @@
     "database",
     "pubsub",
     "realtime",
-    "phpnode"
+    "phpnode",
+    "tracker"
   ],
   "logger": {
     "level": "debug"
   },
+  "tracker": {
+    "segment": ""
+  },
```

Comment:
If we want multiple trackers (or to follow the same format as others), it should be:

```js
"tracker":
  "type": "segment",
  "segment": {
    "secret": "the secret"
  }
```